### PR TITLE
Add missing fields to Image

### PR DESCRIPTION
**Pull request by saffis**
Rebased (follow up for #1260)

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/1345)

### CODE REVIEW

File: docker-java-api/src/main/java/com/github/dockerjava/api/model/Image.java [hunk 1]
```diff
@@ -26,22 +29,38 @@
...
+
+    @JsonProperty("Containers")
+    private Integer containers;
+
     public String getId() {
         return id;
     }
 
-    public String[] getRepoTags() {
+    public List<String> getRepoTags() {
```
Comment (bsideup) [inline docker-java-api/src/main/java/com/github/dockerjava/api/model/Image.java]: please avoid breaking changes on public APIs

Comment (KostyaSha) [reply docker-java-api/src/main/java/com/github/dockerjava/api/model/Image.java]: Contribution doc  mention prefer to use Collections i had idea convert fields, but keep array getter with deprecated mark

Comment (bsideup) [reply docker-java-api/src/main/java/com/github/dockerjava/api/model/Image.java]: @saffis yes, new fields can be `List<String>`, just we can't change the return type of existing getters

**Comment by bsideup** [comment]
@saffis merged 🎉 thank you for your contribution! 👍

**Pull request merged**